principles.md

```diff
@@ -0,0 +1,34 @@
+# Status Principles
+
+The goal of Status is widespread adoption of the decentralized web, also known as web3. Our challenge is achieving mass adoption while staying true to our principles outlined below.
+
+**Liberty**
+We believe in the sovereignty of individuals. As a platform that stands for the cause of personal liberty, we aim to maximize social, political, and economic freedoms.
+
+**Censorship resistance**
+
+We aim for a free flow of information. No content is under surveillance. One of the design goals of Ethereum is to be censorship resistant, and we should provide the same guarantees across Status.
```

Comment:
mentioning `Ethereum` might has similar issue as mentioning `web3`

Reply:
"we abide by the cryptoeconomic design principle of censorship resistance"